chromium_src/third_party/blink/renderer/modules/plugins/dom_plugin_array.cc

```diff
@@ -0,0 +1,59 @@
+/* Copyright (c) 2020 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "brave/third_party/blink/renderer/brave_farbling_constants.h"
+#include "third_party/blink/public/platform/web_content_settings_client.h"
+#include <random>
+
+#define BRAVE_DOM_PLUGINS_UPDATE_PLUGIN_DATA                                   \
+  if (GetFrame() && GetFrame()->GetContentSettingsClient()) {                  \
```

Comment:
is it possible to turn this into a method just so we don't have all the `\`s? Passing in a pointer to `dom_plugins_`?